core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -88,6 +88,7 @@ angular.module('oppia').directive('questionsList', [
           ctrl.getQuestionSummaries =
             QuestionsListService.getCachedQuestionSummaries;
           ctrl.getCurrentPageNumber = QuestionsListService.getCurrentPageNumber;
+          ctrl.isEditorOpen = false;
```

Comment:
Could you make this editorIsOpen, following the existing convention of starting with verbs only for method/function names?

Reply:
Done